data/operators/power/line.json

```
@@ -14,6 +14,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "Východoslovenská distribučná",
+      "locationSet": {"include": ["sk"]},
+      "tags": {
+        "operator": "Východoslovenská distribučná",
+        "operator:short": "VSD",
+        "operator:wikidata": "Q96136456",
+        "power": "substation"
```

```suggestion
        "power": "line"
```
The `power` tag needs to match

---

updated